src/app/modules/item/pages/item-by-id/item-by-id.component.html

```diff
@@ -11,6 +11,7 @@
   <p class="alg-error-message" [style.visibility]="(backToHome.isActive)?'hidden':'visible'">
     <span i18n>Go back to the </span>
     <a
+      class="alg-link base-color"
```

Comment:
The issue is completely addressed, however in [this page](https://dev.algorea.org/branch/feature/links-default-style/en/#/activities/by-id/1121212121212121/details), the not found message style is weird, the top line should have the same font and be centered as well, with also the same font-size.
Considering this PR is very small I think we can fix the style of "Error while loading the item"
![image](https://user-images.githubusercontent.com/16257159/163211387-1f02a93d-bf79-4e4b-b4e7-55c7dad4fb76.png)

Reply:
Fixed